run-test.py

```diff
@@ -413,28 +413,32 @@ def get_finalized_pulse_from_exporter():
     cmd = 'grpcurl -import-path /home/gopher/go/src' +\
           ' -proto /home/gopher/go/src/github.com/insolar/insolar/ledger/heavy/exporter/pulse_exporter.proto' +\
           """ -plaintext localhost:5678 exporter.PulseExporter.TopSyncPulse"""
-    out = get_output("kubectl exec -it jepsen-1 -- /bin/bash -c ' " + cmd + " ' ")
+    out = ssh_output(HEAVY, cmd)
     pulse = json.loads(out)["PulseNumber"]
     info("exporter said: " + str(pulse))
     return pulse
 
 
-def run_benchmark(pod_ips, api_pod=VIRTUALS[0], ssh_pod=1, extra_args=""):
+def run_benchmark(pod_ips, api_pod=VIRTUALS[0], ssh_pod=1, extra_args="", c = C, r = R, timeout = 30):
     virtual_pod_name = 'jepsen-'+str(api_pod)
     port = VIRTUAL_START_PORT + api_pod
-    out = ssh_output(ssh_pod, 'cd go/src/github.com/insolar/insolar && ' +
-                     'timelimit -s9 -t30 ' +  # timeout: 30 seconds
-                     './bin/benchmark -c ' + str(C) + ' -r ' + str(R) + ' -a http://'+pod_ips[virtual_pod_name] +
-                     ':'+str(port) + '/admin-api/rpc ' +
-                     ' -p http://'+pod_ips[virtual_pod_name]+':'+str(port + 100)+'/api/rpc ' +
-                     '-k=./scripts/insolard/configs/ ' + extra_args)
+    out = ""
+    try:
+        out = ssh_output(ssh_pod, 'cd go/src/github.com/insolar/insolar && ' +
+                         'timelimit -s9 -t'+str(timeout)+' ' +
+                         './bin/benchmark -c ' + str(c) + ' -r ' + str(r) + ' -a http://'+pod_ips[virtual_pod_name] +
+                         ':'+str(port) + '/admin-api/rpc ' +
+                         ' -p http://'+pod_ips[virtual_pod_name]+':'+str(port + 100)+'/api/rpc ' +
+                         '-k=./scripts/insolard/configs/ ' + extra_args)
+    except Exception as e:
+        print(e)
+        out = "ssh_output() throwed an exception (non-zero return code)"
```

Comment:
What about added strerr from failed command to this message?

Reply:
OK, sure.